docs/core/tools/dotnet-nuget-trust.md

```diff
@@ -0,0 +1,193 @@
+---
+title: dotnet nuget trust command
+description: The dotnet nuget trust command gets or sets trusted signers to the NuGet configuration.
+author: erdembayar
+ms.date: 06/02/2021
+---
+# dotnet nuget trust
+
+**This article applies to:** ✔️ .NET 5.0.300 SDK, 6.0.100-preview.4 SDK and later versions
+
+## Name
+
+`dotnet nuget trust` - Gets or sets trusted signers to the NuGet configuration.
+
+## Synopsis
+
+```dotnetcli
+dotnet nuget trust [list|author|repository|source|certificate|remove|sync] [OPTIONS]
+
+if none of list|add|remove|sync is specified, the command will default to list.
+
+dotnet nuget trust -h|--help
+```
+
+## Description
+
+The `dotnet nuget trust` gets or sets trusted signers to the NuGet configuration. For additional usage, see [Common NuGet configurations](../../consume-packages/configuring-nuget-behavior.md). For details on how the nuget.config schema looks like, refer to the [NuGet config file reference](../nuget-config-file.md).
+
+## Arguments
+
+- `<list|author|repository|source|certificate|remove|sync>`
+
+  Specifies the command for gets or sets trusted signers to the NuGet configuration. 
+
+#### `> dotnet nuget trust list`
```

Comment:
These should follow the [dotnet sln](https://docs.microsoft.com/dotnet/core/tools/dotnet-sln) example: under an H2 **Commands** each of these should be an h3 with its own set of Synopsis, Arguments, and Options H4 sections.